### ISSUE TITLE
Upgrade to spatie/laravel-permission V5

### ISSUE THREAD

**Issue by jantassov** [vyuldashev/nova-permission#169]
Please, upgrade the library to work with spatie/laravel-permission

**Comment by Freppa** [comment]
following

**Comment by TylerMitton** [comment]
@jantassov is it working at all version 5? what's the issue(s) you're seeing?

**Comment by simonpeters** [comment]
@TylerMitton does your fork work?

**Comment by TylerMitton** [comment]
Yes, it's working, but I'm not using any of the teams functionality.

**Comment by manafn** [comment]
![image](https://user-images.githubusercontent.com/29486466/136032913-1b199845-3ea8-4ae1-baa1-c963f7faa863.png)

**Comment by wagnerdevel** [comment]
+1

**Comment by RomkaLTU** [comment]
+1

**Comment by khanhnam99** [comment]
![image](https://user-images.githubusercontent.com/575720/144181974-5dc0516b-64ed-4b40-9bea-9d286cdf03cd.png)

pls update version "spatie/laravel-permission": "^5.4"

**Comment by chescos** [comment]
Are any changes required to make this package work with `spatie/laravel-permission` v5?

**Comment by robertosanval** [comment]
+1

**Comment by salierg** [comment]
+1

**Comment by parallels999** [comment]
Fixed on https://github.com/vyuldashev/nova-permission/commit/891d1c9e305c19f68ff37e278463643dee4070f1
